### PR TITLE
log.source.hostname changes in log-sources.

### PR DESCRIPTION
**Pull request by badalkumar1223**
## Description
log.source.hostname changes in log-sources.

### CODE REVIEW

File: config/processors/logstash_virtustream_linux_forwarder.conf [hunk 1]
```diff
@@ -8,34 +8,37 @@ input {
...
-    rename => { "[json_tmp][syslog_pid]" => "process.pid" }
-    rename => { "[json_tmp][syslog_severity]" => "log.syslog.priority" }
-    rename => { "[json_tmp][received_from]" => "log.source.ip" }
-    rename => { "[json_tmp][syslog_severity_code]" => "log.syslog.severity.code" }
-    rename => { "[json_tmp][host]" => "host.ip" }
+  mutate {
+    copy => {"event.module" => "log.source.hostname"}
   }
+  # mutate { gsub => ['tmp','[\\]','']}
+  # mutate { gsub => ['source.port','[}]','']}
```
Comment (Akhila-Y) [inline config/processors/logstash_virtustream_linux_forwarder.conf:18]: you can remove the commented block

File: config/processors/logstash_virtustream_linux_forwarder.conf [hunk 2]
```diff
@@ -8,34 +8,37 @@ input {
...
-    rename => { "[json_tmp][@version]" => "observer.version" }
-    rename => { "[json_tmp][message]" => "event.original" }
-    rename => { "[json_tmp][syslog_facility_code]" => "log.syslog.facility.code" }
-    rename => { "[json_tmp][syslog_pid]" => "process.pid" }
-    rename => { "[json_tmp][syslog_severity]" => "log.syslog.priority" }
-    rename => { "[json_tmp][received_from]" => "log.source.ip" }
-    rename => { "[json_tmp][syslog_severity_code]" => "log.syslog.severity.code" }
-    rename => { "[json_tmp][host]" => "host.ip" }
+  mutate {
+    copy => {"event.module" => "log.source.hostname"}
```
Comment (Akhila-Y) [inline config/processors/logstash_virtustream_linux_forwarder.conf:15]: reason why event.module being copied to log.source.hostname??

File: config/processors/api_log_audit_aws.ghe.conf [hunk 3]
```diff
@@ -21,10 +21,13 @@ filter {
     rename => { "[tmp][http_verb]" => "http.request.method" }
     rename => { "[tmp][ident]" => "service.name" }
     rename => { "[tmp][timestamp]" => "event.created" }
-    rename => { "[tmp][host]" => "host.name" }
+    rename => { "[tmp][host]" => "log.source.hostname" }
```
Comment (Akhila-Y) [inline config/processors/api_log_audit_aws.ghe.conf:24]: Badal, could you take updated version and then make the same changes with nested notation?

**Review by Akhila-Y — CHANGES_REQUESTED** [review]

**Comment by KrishnanandSingh** [comment]
Closing this in favor of #194

**Pull request closed without merging**